cfpb/cfgov-refresh
# Consumer Credit Trends data snapshot upgrade for inquiry/denial data

Adds support for new inquiry index and inferred denial data sets for Consumer Credit Trends. Includes stakeholder approval/finalization of template wording.

## Changes

- DataSnapshot Wagtail block
    - Adds fields for inquiry and denial data
    - Updates block test and adds test for correct inquiry/denial display when present
- Management command `update_data_snapshot_values`
    - Supports inquiry and denial data modification for applicable markets
    - Updates management command test and adds test for loading inquiry/denial data
    - Backwards-compatible with prior `data_snapshot.json` files
- Modifies fixture data for DataSnapshot and management command tests
- Includes migration for block changes

## Testing

Management command:
1. Run `tox -e fast` and note no failures occur

Wagtail block:
1. Run `cfgov-refresh` on localhost
1. Run the migration via `manage.py migrate`
1. Visit http://localhost:8000/data-research/consumer-credit-trends/auto-loans/
1. Verify it contains three entries under "Data Snapshot for..." header
1. Edit the following fields in the DataSnapshot block via the admin interface at http://localhost:8000/admin/pages/4477/edit/
    - Inquiry month: `2018-05-01`
    - Inquiry year over year change: `5.6% increase`
    - Inquiry year over year change text: `In year-over-year applications`
    - Denial month: `2018-05-01`
    - Denial year over year change: `5.6% decrease`
    - Denial year over year change text: `In year-over-year unsuccessful applications`
1. Publish your changes
1. Reload http://localhost:8000/data-research/consumer-credit-trends/auto-loans/
1. Verify the Data Snapshot now displays the inquiry and denial entries

## Screenshots

Showing inquiry and denial entries - please note that there should be an image but my local images are currently broken
![Auto loan market page showing inquiry and denial entries](https://user-images.githubusercontent.com/10235730/45891096-aa030680-bd92-11e8-869c-d6c157049fe4.png)

## Checklist
